### PR TITLE
fix a typo for "scope"

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed, added or deleted? (Required)

As in the title

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -4868,7 +4868,7 @@ SHOW WARNINGS;
 > **Note:**
 >
 > - If `tidb_service_scope` is not set for any TiDB node in a cluster, the DXF schedules all TiDB nodes to execute background tasks. If you are concerned about performance impact on current business, you can set `tidb_service_scope` to `background` for a few of the TiDB nodes. Only those nodes will execute background tasks.
```
Comment (qiancai) [inline system-variables.md:4870]: @qiancai

Comment (qiancai) [inline system-variables.md:4870]: @qiancai

**Comment by qiancai** [comment]
@qiancai

**Comment by qiancai** [comment]
this is a test comment @qiancai

**Comment by qiancai** [comment]
this is a test comment2 @qiancai

**Comment by qiancai** [comment]
this is a test commen3 @qiancai

**Comment by qiancai** [comment]
this is a test commen4 @qiancai

File: system-variables.md [hunk 2]
```diff
@@ -4868,7 +4868,7 @@ SHOW WARNINGS;
...
-> - For newly scaled nodes, the DXF tasks are not executed by default to avoid consuming the resources of the scaled node. If you want this scaled node to execute background tasks, you need to manually set `tidb_service_scop` of this node to `background`.
+> - For newly scaled nodes, the DXF tasks are not executed by default to avoid consuming the resources of the scaled node. If you want this scaled node to execute background tasks, you need to manually set `tidb_service_scope` of this node to `background`.
```
Comment (qiancai) [inline system-variables.md:4872]: this is a test commen5 @qiancai

Comment (qiancai) [reply system-variables.md:4872]: this is a test commen6 @qiancai

Comment (qiancai) [reply system-variables.md:4872]: this is a test commen7 @qiancai

Comment (qiancai) [reply system-variables.md:4872]: this is a test commen8 @qiancai

Comment (qiancai) [reply system-variables.md:4872]: this is a test commen9 @qiancai

Comment (qiancai) [reply system-variables.md:4872]: this is a test commen10 @qiancai

Comment (qiancai) [reply system-variables.md:4872]: this is a test commen11 @qiancai

Comment (qiancai) [reply system-variables.md:4872]: this is a test commen12 @qiancai

**Comment by qiancai** [comment]
@qiancai test1920

**Review by dveeden — APPROVED** [review]

**Pull request merged**